exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/ViewHandler.java

```diff
@@ -78,47 +78,58 @@ public PhysicalPlan getPlan(SqlNode sqlNode) throws ValidationException, RelConv
       final View view = new View(newViewName, viewSql, newViewRelNode.getRowType(),
           SchemaUtilites.getSchemaPathAsList(defaultSchema));
 
-      validateViewCreationPossibility(drillSchema, createView, context);
+      final String schemaPath = drillSchema.getFullSchemaName();
```

Comment:
It's a good practice in Java to factor out large code pieces into separate methods. I did this before in this class but your changes incorporated all logic back... Maybe there is a way to keep this logic separately here and in CreateTableHandler.class?

Reply:
Separated the logic as before